src/chrome/content/rules/ReadTheDocs.xml

```diff
@@ -20,11 +23,13 @@
 	
 	<test url="http://testing.readthedocs.org/" />
 	<test url="http://blog.readthedocs.com/" />
+	<test url="http://docs.readthedocs.io/" />
 	<test url="http://testing.rtfd.org/" />
 
 	<securecookie host="^readthedocs\.org$" name=".+" />
 	<securecookie host="^readthedocs\.com$" name=".+" />
+	<securecookie host="^readthedocs\.io" name=".+" />
 	
-	<rule from="^https?://(.+\.)?rtfd\.org/" to="https://$1readthedocs.org/" />
+	<rule from="^https?://(.+\.)?rtfd\.org/" to="https://$1readthedocs.io/" />
```

Comment:
Should this be changed to:

`<rule from="^http://(.+\.)?rtfd\.org/" to="https://$1readthedocs.io/" />`

See https://github.com/EFForg/https-everywhere/issues/1327